src/main/java/com/academia/library/config/OpenApiConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.academia.library.config;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.info.License;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
Lets simplify this:
@Configuration
@OpenAPIDefinition
public class SwaggerConfiguration {
}

And set all what you need in application.yml